README.md

```diff
@@ -1,3 +1,296 @@
-# Helsinki Profile GDPR API Tester
+Helsinki Profile GDPR API Tester
+================================
 
-See [`first`](https://github.com/City-of-Helsinki/profile-gdpr-api-tester/tree/first) branch for a preview version.
+Profile GDPR API Tester is a command line tool for testing GDPR API implementations that conform to the Helsinki Profile GDPR API specification.
+This tools main usage is to test the API Token authentication in the implemented GDPR API.
+
+See [GDPR API Description](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/KAN/pages/80969736/GDPR+API+Description) for more information about the GDPR API.
+
+# Requirements
+
+Python 3.10+ or Docker
+
+# Installing / Running
+
+## Running from a GitHub package using Docker
+
+Copy [env.example](env.example) to your local filesystem as `.env` and change the settings to suit your environment. (See [configuration](#configuration))
+
+```shell
+docker run -i -p 8888:8888 --env-file .env ghcr.io/city-of-helsinki/profile-gdpr-api-tester
+```
+
+
+## Installing using pip
+
+**TODO**
```

Comment:
What this `TODO` is about? What needs to be done?

Reply:
Ah. That was because the package is not yet in PyPI. I removed it as the package could be in PyPI soon.